buzzmobile/constants.yaml

```diff
@@ -13,7 +13,12 @@ pixels_per_m: 80
 # bearing calculation accuracy.
 median_filter_size: 5
 
-# Used for tentacle projection, travel distance is the step between each 
+# Minimum distance a positional fix must be from the previous
+# fix in order to add its bearing to the median filter. Used to
+# improve bearing accuracy at low speeds.
+min_fix_distance: 0.1
```

Comment:
Also, depending on the units this might be too large a distance, no? If it's radians, then the min distance is enormous for a sphere the size of the earth.